components/constellation/constellation.rs

```diff
@@ -1807,11 +1811,11 @@ impl<Message, LTF, STF> Constellation<Message, LTF, STF>
         };
 
         let child_pipeline_ids: Vec<PipelineId> = self.full_frame_tree_iter(frame_id)
-                                                      .flat_map(|frame| frame.next.iter()
-                                                                .chain(frame.prev.iter())
-                                                                .chain(once(&frame.current)))
-                                                      .map(|state| state.pipeline_id)
-                                                      .collect();
+            .flat_map(|frame| frame.prev.iter().chain(frame.next.iter())
```

Comment:
ahhh, didn't realize this was all inside `flat_map`... I was confused what `frame` was even referencing 2 lines below.